### ISSUE TITLE
Cryptic error messages when providing a config file without the expected section

### ISSUE THREAD

**Issue by tovrstra** [PyCQA/pydocstyle#276]
This accidentally happened to me and is easy to reproduce. Do the following in an empty directory, using pydocstyle 2.0.0:

```bash
echo '[pydcstyle]' > config
touch test.py
pydocstyle test.py --config=config
```

Then I get the following error message:
```
Traceback (most recent call last):
  File "/home/toon/.local/bin/pydocstyle", line 11, in <module>
    sys.exit(main())
  File "/home/toon/.local/lib/python2.7/site-packages/pydocstyle/cli.py", line 68, in main
    sys.exit(run_pydocstyle())
  File "/home/toon/.local/lib/python2.7/site-packages/pydocstyle/cli.py", line 43, in run_pydocstyle
    conf.get_files_to_check():
  File "/home/toon/.local/lib/python2.7/site-packages/pydocstyle/config.py", line 171, in get_files_to_check
    config = self._get_config(name)
  File "/home/toon/.local/lib/python2.7/site-packages/pydocstyle/config.py", line 262, in _get_config
    config = self._create_check_config(options)
  File "/home/toon/.local/lib/python2.7/site-packages/pydocstyle/config.py", line 383, in _create_check_config
    if cls._has_exclusive_option(options) or use_defaults:
  File "/home/toon/.local/lib/python2.7/site-packages/pydocstyle/config.py", line 497, in _has_exclusive_option
    cls.BASE_ERROR_SELECTION_OPTIONS])
AttributeError: 'NoneType' object has no attribute 'ignore'
```
It is too difficult to figure out what this error message means.

**Comment by Nurdok** [comment]
Thanks for reporting this @tovrstra. I agree there should be a better error message.

**Comment by FarmerSez** [comment]
Implemented in #280 .

**Comment by tovrstra** [comment]
Thanks!
